app.py

```diff
@@ -72,11 +73,15 @@ class Meta:
 #create a User
 @app.route('/user', methods=['POST'])
 def create_user():
+    data = request.get_json()
+    if 'first_name' not in data or 'last_name' not in data or 'email' not in data or 'password' not in data or 'phone_number' not in data:
+        return bad_request('Error: Missing Fields')
+    if User.query.filter_by(email=data['email']).first():
```

Comment:
@JellyBeans1312 @gregoryanderson this restricts users from creating an account with an email that's already stored in the database; were we talking about this yesterday?